### DISCUSSION TITLE
[Bug] Hiding and showing the body doesn't work properly

### DISCUSSION THREAD

**Discussion by Igcorreia** [webstudio-is/webstudio-community#46] [Q&A]
I was testing the property panel, and I disabled and then enabled the body. Now the body lives below Grammarly or other extensions. 

I think <body> should not be possible to disable unless we want the body to be injected by javascript or when it is the body we prepend instead of appending. 

![image](https://github.com/webstudio-is/webstudio/assets/1076409/bf7a5af1-6d1f-48ab-a122-765f976be069)
![image](https://github.com/webstudio-is/webstudio/assets/1076409/396834b3-0814-4cf8-9eb6-781997ee5744)

**Comment by kof** [comment]
Oh damn, body shouldn't be hidable cc @TrySound
